now.json

```
@@ -187,6 +187,11 @@
       "status": 301,
       "headers": { "Location": "https://github.com/zeit/now-examples" }
     },
+    {
+      "src": "/docs/v1/examples/(.*)",
```

Don't need to capture the group since we're not using the information. Just slightly faster on the network side of things too.
```suggestion
      "src": "/docs/v1/examples/.*",
```